src/styles/blocks/catalog.scss

```diff
@@ -18,6 +21,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: color 0.3s, transform 0.3s;
```

Comment:
Create a variable for transition duration, as it should be the same for the whole page and is frequently used